src/framework/components/element/text-element.js

```diff
@@ -733,6 +733,17 @@ class TextElement {
                                 break;
                             }
                         }
+
+                        // #ifdef DEBUG
+                        if (!json.missingChars) {
+                            json.missingChars = new Set();
+                        }
+
+                        if (!json.missingChars.has(char)) {
+                            console.error("Character ", char, "is missing in the font");
```

Comment:
This should probably be:
```suggestion
                            console.warn("Character '", char, "' is missing from the font");
```

Reply:
thanks, i updated the pr